index.html

```diff
@@ -66,99 +76,103 @@ <h2>My Recent Works</h2>
       </div>
       <div class="projects">
         <img src="images/mainPH.svg" alt="img placeholder" />
-        <h2>Multi-Post Stories</h2>
-        <p>
-          A daily selection of privately personalized reads; no accounts or
-          sign-ups required. has been the industry's standard dummy text ever
-          since the 1500s, when an unknown printer took a standard dummy text.
-        </p>
-        <ul class="tools">
-          <li><button class="toolsBtn">CSS</button></li>
-          <li><button class="toolsBtn">HTML</button></li>
-          <li><button class="toolsBtn">Bootstrap</button></li>
-          <li><button class="toolsBtn">Ruby</button></li>
-        </ul>
-        <button class="mainSeeBtn">See Project</button>
-      </div>
```

Comment:
I've reviewed your design and noticed that it doesn't quite match the Figma template. The main issue I see is that there are differences in the font sizes, widths, heights, margins, and paddings. Could you please take a closer look and adjust these elements to match the template more closely?


**I am attaching screenshots of both designs to make them more clear! 👍** 

## Your Design:-

![image](https://user-images.githubusercontent.com/94223367/220548488-18d02008-671f-4fee-8510-0ab831ab4133.png)

## Figma Design:-

![image](https://user-images.githubusercontent.com/94223367/220548592-b1fb284c-ad84-4f3c-8708-1ce555e2d6c3.png)


## Remember:-

You did great work but the project demands to make it as close as possible to the design almost 100% exact. So your design font-sizes, margins, padding, width, and height are actually similar to the design.